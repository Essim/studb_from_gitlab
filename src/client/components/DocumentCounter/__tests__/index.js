import React from 'react';
import {DocumentCounter} from '../';
import { shallowToJson } from 'enzyme-to-json';

const mockCounter = "like",
    mockItems = ["mockItem1","mockItem2","mockItem3"];

describe('DocumentCounter', () => {
    it('should render correctly', () => {
        const output = shallow(<DocumentCounter counter={mockCounter} items={mockItems}/>);
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});
