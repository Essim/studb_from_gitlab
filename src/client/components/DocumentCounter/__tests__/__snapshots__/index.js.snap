// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DocumentCounter should render correctly 1`] = `
<LikeCounter
  items={
    Array [
      "mockItem1",
      "mockItem2",
      "mockItem3",
    ]
  }
/>
`;
